src/components/Workspace.jsx

```diff
@@ -62,6 +72,9 @@ class Workspace extends React.Component {
       '_handleClickInstructionsBar',
       '_handleComponentUnhide',
       '_handleComponentHide',
+      '_handleOutputDividerDrag',
+      '_handleOutputDividerStart',
+      '_handleOutputDividerStop',
```

Comment:
The `<Workspace>` component is basically a big ball of legacy code—it used to be the _only_ connected component in the entire application, requiring all data to flow through it, making it an even bigger mess than it is today.

I started a big refactor to add containers (connected components), although I didn’t entirely finish. In particular, the `<EditorsColumn>` should have a container, but doesn’t. The end goal of the refactor would be for `<Workspace>` to be a purely presentational component (probably with its own container to handle a couple of things).

In general I would like to avoid _adding_ new data flow paths to `<Workspace>`, as it brings us further from the goal of being purely presentational. I generally make an exception for things that need to support the `<EditorsColumn>`, because that’s just an unfinished bit of the container restructuring. But in this case, we’re actually supporting `<Output>`, which currently doesn’t have a container simply because it hasn’t needed one`*`. But here it looks like that is no longer the case.

So, I think we should introduce an `Output` container, and have the drag handling flow through it to the Redux store.

`*` Incidentally, there are a couple of other bits of logic that would benefit from an `Output` container—for instance, the question of whether to display the `<Console>` at all fits more naturally in `<Output>` rather than `<Console>` itself, but was more convenient to deal with in `<Console>` since the latter is a connected component. No need to make those changes as part of this PR but making an `Output` container would afford some additional code improvements.

Reply:
Yes! I can go ahead and containerize the Output.